mage_ai/io/qdrant.py

```diff
@@ -0,0 +1,151 @@
+from typing import List
+
+from pandas import DataFrame
+from qdrant_client import QdrantClient
+from qdrant_client.http import models
+from sentence_transformers import SentenceTransformer
+
+from mage_ai.io.base import BaseIO
+from mage_ai.io.config import BaseConfigLoader, ConfigKey
+
+
+class Qdrant(BaseIO):
+    def __init__(
+            self,
+            collection: str,
+            path: str = None,
+            verbose: bool = True,
+            **kwargs,) -> None:
+        """
+        Initializes connection to qdrant db.
+        """
+        super().__init__(verbose=verbose)
+        self.collection = collection
+        if path is None:
+            self.client = QdrantClient(':memory:')
+        else:
+            self.client = QdrantClient(path=path)
+
+    @classmethod
+    def with_config(cls, config: BaseConfigLoader) -> 'Qdrant':
+        return cls(
+            collection=config[ConfigKey.QDRANT_COLLECTION],
+            path=config[ConfigKey.QDRANT_PATH],
+        )
+
+    def create_collection(
+            self,
+            vector_size: int,
+            distance: models.Distance = None,
+            collection_name: str = None):
+        """
+            Create collection in qdrant db.
```

Comment:
Can you remove the extra indent in the comments?

Reply:
Updated